draft/ZEP0001.md

```diff
@@ -228,23 +228,19 @@ following specifications have also been published on the zarr-specs
 website:
 
 * [Codecs](https://zarr-specs.readthedocs.io/en/latest/codecs.html)
```

Comment:
Link might need an update when https://github.com/zarr-developers/zarr-specs/pull/187 is merged.